Pod/Classes/PINRemoteImageDownloadTask.m

```diff
@@ -42,6 +42,7 @@ - (void)callProgressWithQueue:(dispatch_queue_t)queue withImage:(UIImage *)image
     }];
 }
 
+#warning The manager is not used in here, just dropped by a warning to make sure the problem is tackled in the future
```

Comment:
Do you mind filing a task instead? I'll take care of it there instead of adding a warning to the project.